documents/charter.md

```diff
@@ -1,17 +1,19 @@
-Video traffic is already 70% of the overall traffic volume on the Internet and is expected to grow to 80% by 2028.
+Video traffic is 70% of the overall traffic volume on the Internet and is expected to grow to 80% by 2028.
 Across developed and emerging markets video traffic forms 50-80% of traffic volume on mobile networks.
 New formats like short form videos have seen tremendous growth in recent years.
 These growth trends are likely to increase with new populations coming online on mobile-first markets.
 
-Mobile network operators continuously invest in network resources, including deployment of new generations or new bands of spectrum.
-Since spectrum is a limited and expensive resource, operators often make use of flow-based traffic handling such as shaping of video traffic, especially when the network is highly loaded.
-Operators cannot explicitly measure the degradation that shaping causes to end user quality of experience (QoE), making this approach open loop. 
+Local mobile radio conditions may constrain the maximum throughput for a given client, or be so volatile as to rapidly change the maximum throughput throughout the course of a session. 
+In addition, despite capacity augmentation work such as deployment of new generations or new bands of spectrum, capacity augmentation efforts are not keeping pace with growth in demand. 
+These network operators have found it faster and less expensive to invest in shaping (also called throttling) of video traffic on a per-flow basis, which negatively affects video stream quality. 
+This is done for both network management and business motivations.
+Because so much of this video traffic is encrypted, network operators cannot explicitly measure the degradation to end user quality of experience (QoE) caused by traffic shaping, making this approach open loop. 
```

Comment:
Encrypted or not, it would be very difficult for a network operator to explicitly measure degradation of QoE. 
I would suggest that we remove the first part of the sentence and start with "Network operators .."